.pre-commit-config.yaml

```diff
@@ -0,0 +1,17 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 21.9b0
+    hooks:
```

Comment:
Last comment: I would remove this again. It does not provide any value for the content

Reply:
@dynox please remove this